transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessengerTransport.kt

```diff
@@ -58,4 +56,8 @@ class MessengerTransport(private val configuration: Configuration, private val t
     suspend fun fetchDeploymentConfig(): DeploymentConfig {
         return DeploymentConfigUseCase(configuration.logging, configuration.deploymentConfigUrl.toString()).fetch()
     }
+
+    companion object {
```

Comment:
Minor. Can we have the TOKEN_STORE_KEY as a top level declaration instead of  companion object? 
Also, we can remove the `tokenStoreKey` from Configuration.kt